src/static/js/app/survey.js

```diff
@@ -4,4 +4,4 @@ import ReactDOM from 'react-dom';
 
 import Survey from './components/Survey';
 
-ReactDOM.render(<Survey />, document.getElementById('survey'));
+ReactDOM.render(<Survey quizType={window.quizType} />, document.getElementById('survey'));
```

Comment:
Line is too long.<br>Expected an identifier and instead saw '<'.<br>Expected ')' and instead saw 'Survey'.<br>Missing semicolon.<br>Expected ':' and instead saw '.'.<br>Unclosed regular expression.<br>Unrecoverable syntax error. (87% scanned).